.github/workflows/ci-wheels.yml

```diff
@@ -35,9 +35,7 @@ jobs:
     - name: "building"
       shell: bash
       run: |
-        # require build with explicit --sdist and --wheel in order to
-        # get correct version associated with sdist and bdist artifacts
-        pipx run build --sdist --wheel
```

Comment:
I've checked to confirm the the resultant `sdist` and binary `wheel` generated from `python -m build` and `pipx run build` both result in artifacts that have the correct version